internal/locate/region_request.go

```diff
@@ -1548,13 +1655,10 @@ func (s *RegionRequestSender) onRegionError(bo *retry.Backoffer, ctx *RPCContext
 		return retry, err
 	}
 
-	if regionErr.GetServerIsBusy() != nil {
-		// Mark the server is busy (the next incoming READs could be redirect
-		// to expected followers. )
-		if ctx != nil && ctx.Store != nil {
-			ctx.Store.markAlreadySlow()
+	if serverIsBusy := regionErr.GetServerIsBusy(); serverIsBusy != nil {
+		if s.replicaSelector != nil {
```

Comment:
If the `replicaSelector` is nil would the `markAlreadySlow` be skipped unexpectedly?

Reply:
Actually, this is impossible for TiKV requests. `RegionRequestSender` always creates a new `replicaSelector` if not exists.